src/index.scss

```diff
@@ -1,4 +1,4 @@
-@import './styles/variables';
+@import './components/styles/variables';
```

Comment:
Let's move the `src/components/styles` directory back into `src/styles`. We should be treating it like the `src/img` directory.